content/applications/inventory_and_mrp/manufacturing/plm/version_control.rst

```diff
@@ -0,0 +1,142 @@
+===============
+Version control
+===============
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Material)`
+.. |BOMs| replace:: :abbr:`BoMs (Bill of Materials)`
+.. |ECO| replace:: :abbr:`ECO (Engineering change order)`
+.. |ECOs| replace:: :abbr:`ECOs (Engineering change orders)`
```

Comment:
Since this is the first PLM documentation, I'm not sure there's an existing fashion to write "ECO" in... to keep consistent with the acronyms, do you think we should go with:
-ECO: Engineering Change Order
-ECO: engineering change order

or, the way you have it: Engineering change order ... ?

Worth thinking about and setting a precedent, I think, but totally up to you